src/Sulu/Bundle/PageBundle/Resources/phpcr-migrations/Version202005191116.php

```diff
@@ -49,6 +56,7 @@ private function upgrade(SessionInterface $session)
         $rows = $queryManager->createQuery($query, 'JCR-SQL2')->execute();
 
         foreach ($rows as $row) {
+            /** @var NodeInterface $node */
```

Comment:
Can be null only on `outer join` which we don't use so use phpdoc to overwrite the `NodeInterface|null` in the migration